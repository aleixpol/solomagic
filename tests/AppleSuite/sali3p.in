\ref something0
\mt somethingsomething salindi somethingsomething

\ref something1
\mt somethingsomething salad somethingsomething

