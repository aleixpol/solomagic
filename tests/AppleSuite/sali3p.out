\ref something0
\mt somethingsomething salindi somethingsomething
\ma somethingsomething salingri somethingsomething

\ref something1
\mt somethingsomething salad somethingsomething
\ma somethingsomething salad somethingsomething
